FUSETOOLS-2477: Adapt QE tests to RedDeer 2.0 changes

apodhrad opened the pull request.

apupier commented on pom.xml:
```diff
@@ -98,6 +98,20 @@
 			<url>${jbosstools-central-site}</url>
 		</repository>
 
+		<!-- Remove it once RedDeer 2.0.0 is on targetplatform -->
```
do you know when it is planned to be added?

apupier commented on pom.xml:
```diff
@@ -97,6 +98,20 @@
+			<url>http://download.eclipse.org/reddeer/milestones/2.0.0.Beta/</url>
+		</repository>
+
+		<!-- Remove it once https://github.com/eclipse/reddeer/pull/1803 is merged -->
```
it has been merged :-)

apodhrad commented:
Hi @apupier, the PR is ready

apupier commented on qe/README.md:
```diff
@@ -0,0 +1,65 @@
+
+Current version of JBoss Tools Fuse is built on Eclipse Oxygen, so download Eclipse Oxygen for JEE Developers. For executing the tests from IDE you need
+
+ - Target Platform (targetplatform/multiple)
```
it is now just targetplatform

apupier commented on qe/plugins/org.jboss.tools.fuse.qe.reddeer/.classpath:
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7">
```
all projects in Fuse Tooling (and in Eclipse in general) are now 1.8

apodhrad replied:
nice catch! I need to update it in all occurrences

apupier commented on qe/plugins/org.jboss.tools.fuse.qe.reddeer/.settings/org.eclipse.jdt.core.prefs:
```diff
@@ -0,0 +1,8 @@
+org.eclipse.jdt.core.compiler.problem.assertIdentifier=error
+org.eclipse.jdt.core.compiler.problem.enumIdentifier=error
+org.eclipse.jdt.core.compiler.problem.forbiddenReference=warning
+org.eclipse.jdt.core.compiler.source=1.7
```
all projects in Fuse Tooling (and in Eclipse in general) are now 1.8

apupier commented on qe/plugins/org.jboss.tools.fuse.qe.reddeer/META-INF/MANIFEST.MF:
```diff
@@ -0,0 +1,37 @@
+ org.jboss.tools.fuse.qe.reddeer.wizard
+Bundle-Vendor: %Bundle-Vendor
+Import-Package: com.fasterxml.jackson.annotation
+Eclipse-BundleShape: dir
```
Out of curiosity, why is it BundleShape dir?

apodhrad replied:
Because we copy some files from that bundle and it is easier to copy from dir than from jar

apupier commented on qe/tests/org.jboss.tools.fuse.qe.reddeer.tests/launcher/RedDeer Test Linux x86_64.launch:
```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
shouldn't we named it RedDeer Test? it is taking the whole set of available plugins so no difference on the platform unless I missed a configuration?

apodhrad replied:
Good point! This was changed in recent commit and I didn't notice that :( I will prepare another launch config.

apodhrad commented:
Hi @apupier, I have updated the PR according to your suggestions. Regarding the naming I have raise an issue at https://issues.jboss.org/browse/FUSETOOLS-2537 where we can discuss the name.

apodhrad commented:
Regarding the launch config I have deleted the Linux-based one (at the moment I'm not able to use the target platform so I can't choose the correct bundles). The existing one I renamed to 'RedDeer Test' which takes all bundles in a workspace.

Moreover, I extended the README file by a section about how to run tests without a target platform since working with the target platform is a big pain :(

apupier commented:
testPR

apupier commented:
testPR

apodhrad commented:
merged by #1109
